packages/teleport/src/config.ts

```diff
@@ -29,7 +29,7 @@ import generateResourcePath from './generateResourcePath';
 
 const cfg = {
   // TODO(isaiah): remove after feature is finished.
-  enableDirectorySharing: false, // note to reviewers: should be false in any PRs.
+  enableDirectorySharing: true, // note to reviewers: should be false in any PRs.
```

Comment:
This feels like one of those [logic puzzles](https://en.wikipedia.org/wiki/The_Hardest_Logic_Puzzle_Ever)… 🤔

But in all seriousness, is it supposed to be true or false in the end? 😅

Reply:
Haha

You got it correct, its supposed to be `false` in the end. Fixed.